Improve the README on the wl.org CloudFront redirects (#273)
wellcomecollection/platform-infrastructure: cloudfront/wellcomelibrary.org/README.md

    As of January 2022, we are not redirecting OPAC URLs, and we have no immediate plans to do so.

*   DServe was a web front-end for the archive catalogue/CALM, available at `archives.wellcomelibrary.org`.
    We redirect requests to the Works pages on the new website.

[opac]: https://en.wikipedia.org/wiki/Online_public_access_catalog

## Getting to the Route 53 Hosted Zone

The Route 53 Hosted Zone for wellcomelibrary.org is defined in a D&T account.

You can get to the Hosted Zone by assuming the following role: `arn:aws:iam::267269328833:role/wellcomecollection-assume_role_hosted_zone_update`.

![Screenshot of the "assume role" screen in the AWS console with the account ID and role filled in.](assume_role.png)

You can see the Hosted Zone by going to <https://console.aws.amazon.com/route53/v2/hostedzones?#ListRecordSets/Z78J6G8RSOLSZ>

(You can't find the hosted zone in the Route 53 console because we don't have the ListHostedZones permission – some of them are for domains we don't control.)

## How to deploy the redirects

> 1.  Create a ZIP package for the Lambda definition, which is uploaded to S3:

Reviewer: Imperative mood throughout
```suggestion
1.  Package and upload to S3 a ZIP for the Lambda definition
```
